src/components/games/GameControlsMod.svelte

```diff
@@ -197,11 +231,26 @@
         placement="top-end"
         class="!bg-slate-900 overflow-y-auto max-h-[300px]"
       >
+        <div class="p-2">
+          <label class="flex items-center space-x-2">
+            <input
+              type="checkbox"
+              class="form-checkbox text-orange-500 bg-slate-700 rounded focus:ring-orange-500"
+              id="autoUpdate"
+              checked={autoUpdateChecked}
+              on:change={toggleAutoUpdate}
+            />
+            <span class="text-sm text-white font-semibold"
+              >{$_("gameControls_get_newest_version")}</span
+            >
+          </label>
+        </div>
```

Comment:
use flowbite components, not default HTML ones https://flowbite-svelte.com/docs/components/dropdown#Dropdown_with_checkbox

```svelte
<DropdownItem>
          <Checkbox color="orange" checked={autoUpdateChecked} on:change={toggleAutoUpdate}>{$_("gameControls_get_newest_version")}</Checkbox>
        </DropdownItem>
```